### DISCUSSION TITLE
Time Format only change date in Result ?

### DISCUSSION THREAD

**Discussion by masterwishx** [alexjustesen/speedtest-tracker#291] [Q&A]
Changed Time format to ( j M, Y H:i:s ) but changes only visible in Result but not in Dashboard.
is it OK?

**Comment by alexjustesen** [comment]
Not sure I follow, can you attach a screenshot?

**Comment by masterwishx** [comment]
Sure


![image](https://user-images.githubusercontent.com/28630321/210733767-89a9ed6f-064e-4d74-a9ce-3b6b01218d91.png)

Here changed to (j M, Y H:i:s)

![image](https://user-images.githubusercontent.com/28630321/210733652-63f99eb8-0e75-4a6e-b094-f7b364a401a3.png)

but here still (M d, Y H:i)

![image](https://user-images.githubusercontent.com/28630321/210733528-7b68236b-61f1-491c-9525-9654fb240a07.png)

**Reply by alexjustesen** [reply]
Oh I did that intentionally because having long dates in the chart looked horrible

**Reply by masterwishx** [reply]
OK got it, but for me more regular to see day befor month. So if will be option to change it will be very cool. Thanks

**Reply by alexjustesen** [reply]
I can split out the setting to a chart date so it can be passed to those components. Not going to be a high priority but something I can sneak in when I'm working on charting again.

**Reply by alexdelprete** [reply]
@masterwishx if you like graphs, use Grafana, ST already exports to Influx, and I prepared a dashboard for this project. I don't know if the instructions are already in the docs yet but it's easy to configure it. So you will have total control of data and how it's being displayed (in Grafana).

**Reply by masterwishx** [reply]
> I can split out the setting to a chart date so it can be passed to those components. Not going to be a high priority but something I can sneak in when I'm working on charting again.

Thanks, will be great

**Reply by masterwishx** [reply]
> if you like graphs, use Grafana, ST already exports to Influx

@alexdelprete Thanks, I already using grafana for this project, and also having dashboard for it, Also posted grafana dashboard for old project and will post for this when will have time.
Also checked your dashboard,its very cool and Thanks for you work
